recipes/go4/recipe.yaml

```diff
@@ -0,0 +1,94 @@
+context:
+  name: Go4
+  version: '6.4.1'
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/gsi-ee/go4//releases/download/${{ version }}/go4-${{ version }}.tar.gz
+  sha256: d4c4409b4a86e8686ac19a040824fc37f8ad9bdeef27545cb405b777025e12ee
+
+build:
+  number: 0
+  skip:
+    - win
+    - osx
+  script:
+    - if: unix
+      then:
+      - "${RECIPE_DIR}/build.sh"
+      - "${RECIPE_DIR}/clean_up.sh"
+  dynamic_linking:
+    overdepending_behavior: "error" # (defaults to "ignore")
+    overlinking_behavior: "error" # (defaults to "ignore")
+
+requirements:
+  build:
+    - cmake
+    - make
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - ${{ compiler('cxx') }}
+    - libgl-devel
+  host:
+    # conda-forge pins root_base 6.32 and 6.34 as variants
+    # however root_base 6.32 is pinned to gcc version 13, but gcc is pinned to 14
+    - root_base 6.34.*
+    - qt
+    - hdf5
+    - python
+  run:
+    - python
+
+tests:
+  - package_contents:
+      bin:
+      - go4
+      - go4analysis
+  - script:
+      - ./run_go4_examples.sh
+    files:
+      source:
+        - Go4ExampleSimple/
+        - Go4ExampleUserSource/
+      recipe:
+        - run_go4_examples.sh
+    requirements:
+      run:
+      - cmake
+      - make
+      # works:
+      - gxx
+      # But:
+      # - ${{ compiler('cxx') }}
+      # Doesn't work: 
+      # │ make: g++: No such file or directory
+      # │ make: *** [$PREFIX/Makefile.rules:50: TXXXProc.o] Error 127
```

Comment:
If you need `g++`, feel free to add also the `cxx-compiler` output just for the test.

Reply:
`${{ compiler('cxx') }}` resolves to `gxx_linux-64 14.*`. That one apparently does not include `g++`.
I just expected to get it from the compiler cxx macro, as that worked for the build.

Reply:
But something that I do not understand, why the example requires `g++` ? Shouldn't it observe the `CXX` env variable as done by most build systems?